src/components/Stepper/index.tsx

```diff
@@ -80,7 +81,8 @@ const StyledTouchable = styled(Touchable, {
   margin: 0 4px;
 `
 
-const StyledCenterTouchable = styled(Touchable)<{ size: ContainerSizesType }>`
+const StyledCenterBox = styled(Box)<{ size: ContainerSizesType }>`
```

Comment:
Why use a Box ?

Reply:
I was about to comment the same. Any reason @matthprost ?